### PR TITLE
Enhance patch/test cases

### PR DESCRIPTION
**Pull request by hantmac**
- Enrich test cases.

### CODE REVIEW

File: schema/field/field_test.go [hunk 1]
```diff
@@ -52,6 +57,8 @@ func TestFloat(t *testing.T) {
 	f = field.Float("age").Min(2.5).Max(5)
 	fd = f.Descriptor()
 	assert.Len(t, fd.Validators, 2)
+
```
Comment (a8m) [inline schema/field/field_test.go]:
```suggestion
```

File: schema/field/field_test.go [hunk 2]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
 	assert.NotNil(t, fd.Default)
 	assert.NotEmpty(t, fd.Default.(func() uuid.UUID)())
 }
+
+func TestTypeString(t *testing.T) {
+	var typ = field.TypeBool
```
Comment (a8m) [inline schema/field/field_test.go]:
```suggestion
	typ := field.TypeBool
```

File: schema/field/field_test.go [hunk 3]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
...
+	assert.Equal(t, "bool", typ.String())
+	typ = field.TypeInvalid
+	assert.Equal(t, "invalid", typ.String())
+	typ = 21
+	assert.Equal(t, "invalid", typ.String())
+}
+
+func TestTypeNumeric(t *testing.T) {
+	var typ = field.TypeBool
```
Comment (a8m) [inline schema/field/field_test.go]:
```suggestion
	typ := field.TypeBool
```

File: schema/field/field_test.go [hunk 4]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
...
+	assert.Equal(t, false, typ.Numeric())
```
Comment (a8m) [inline schema/field/field_test.go]:
```suggestion
	assert.False(t, typ.Numeric())
```

File: schema/field/field_test.go [hunk 5]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
...
+	typ = field.TypeUint8
+	assert.Equal(t, true, typ.Numeric())
```
Comment (a8m) [inline schema/field/field_test.go]:
```suggestion
	assert.Equal(t, true, typ.Numeric())
```
```suggestion
	assert.True(t, typ.Numeric())
```

File: schema/field/field_test.go [hunk 6]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
...
+}
+
+func TestTypeValid(t *testing.T) {
+	var typ = field.TypeBool
+	assert.Equal(t, true, typ.Valid())
+	typ = 0
+	assert.Equal(t, false, typ.Valid())
```
Comment (a8m) [inline schema/field/field_test.go]: ditto

File: schema/field/field_test.go [hunk 7]
```diff
@@ -164,3 +171,35 @@ func TestField_UUID(t *testing.T) {
...
+}
+
+func TestTypeConstName(t *testing.T) {
+	var typ = field.TypeJSON
```
Comment (a8m) [inline schema/field/field_test.go]: ditto

**Comment by hantmac** [comment]
@a8m Thanks for your suggestions.

File: schema/field/field_test.go (hunk 7 above)
Comment (hantmac) [reply schema/field/field_test.go]: I should follow the code style in  `ent`.

**Pull request merged**